src/widgets/settingspages/GeneralPageView.cpp

```diff
@@ -291,22 +291,30 @@ bool GeneralPageView::filterElements(const QString &query)
 
             QWidget *currentSubtitle = nullptr;
             bool currentSubtitleVisible = false;
+            bool currentSubtitleSearched = false;
 
             for (auto &&widget : group.widgets)
             {
-                if (dynamic_cast<SubtitleLabel *>(widget.element))
+                if (auto x = dynamic_cast<SubtitleLabel *>(widget.element))
```

Comment:
Sorry for nitpicking, but please change the variable's name to something more explanatory.

Reply:
https://github.com/Chatterino/chatterino2/blob/0a51252b761ac4a846725ce6b6e8cf8bba5efdf4/src/widgets/settingspages/GeneralPageView.cpp#L266
I was using the same naming scheme as the code above.

Reply:
Okay, I guess sticking to the consistency with code above makes sense then :ok_hand: I was just a little confused myself while reading this but it's ok as the variable is only used once.